debian/rules

```diff
@@ -32,10 +32,9 @@ ifeq ($(filter noopt,$(DEB_BUILD_OPTIONS)),noopt)
 	DEB_BUILD_MAINT_OPTIONS += optimize=-lto
 endif
 
-# Disable LTO on jammy due to failing to build
+# Use ldd on jammy due to failing to build with ld
 ifneq ($(filter jammy,$(DEB_DISTRIBUTION)),)
```

Comment:
```suggestion
ifeq ($(DEB_DISTRIBUTION),jammy)
```

Reply:
Makes sense, but I wanted to only touch the minimum.